ui/src/app/app-routing.module.ts

```diff
@@ -125,6 +126,14 @@ const routes: Routes = [
                       breadcrumb: BreadcrumbType.Notebook,
                       minimizeChrome: true
                     }
+                  }, {
+                    path: 'v2/:nbName',
```

Comment:
nit: I might just call this `preview` rather than v2

Reply:
does `preview` make sense as more features like edit and playground mode get added?

Reply:
Could go either way, putting "edit" / "playground" in the URL may make sense. Alternatively if at some point we move the Jupyter editor to this page and deprecate the current notebooks redirect page, we could rename this back to just "notebooks" and delete the other path.

Reply:
yeah. I was thinking we'll eventually move back to just `notebooks`